rest-service/manager_rest/security/secured_resource.py

```diff
@@ -97,12 +109,22 @@ def premium_only(f):
         """
         return f
 else:
-    premium_only = missing_premium_feature_abort
+    premium_only = _abort_on_premium_missing
+
+
+def allow_on_community(func):
```

Comment:
Maybe add two simple tests for the decorators

Reply:
no useful way to do it, would have to go through the actual request cycle, and all the existing tests already do that
(because `MissingPremiumFeatureResource` doesn't actually _use_ `method_decorators` unless it's serving a request)